src/styles/globals.scss

```diff
@@ -6,6 +6,11 @@
   margin: 0;
 }
 
+html {
+  scroll-padding-top: $navbarHeight;
```

Comment:
Wydaje mi się, że tą wartość można dodać do offsetu w komponencie i zamknąć implementacje scrolla tylko w nav

Reply:
Ten properties jest potrzebny, żeby zmienić zachowanie scrolla, gdy klikamy jakiś link w navie - bez niego przy scrollowaniu nav przykrywa część sekcji - patrze screen niżej

![wtf](https://user-images.githubusercontent.com/20808724/151675287-597eb78f-0152-43e0-b5be-61fc6a88a82b.png)

jeśli masz pomysł jak to rozwiązać bez tego propertiesa z offsetem - to chętnie się dowiem więcej ;)

Reply:
Ten properties jest potrzebny, żeby zmienić zachowanie scrolla, gdy klikamy jakiś link w navie - bez niego przy scrollowaniu nav przykrywa część sekcji - patrze screen niżej

![wtf](https://user-images.githubusercontent.com/20808724/151675287-597eb78f-0152-43e0-b5be-61fc6a88a82b.png)

jeśli masz pomysł jak to rozwiązać bez tego propertiesa z offsetem - to chętnie się dowiem więcej ;)